### PR TITLE
create explicit routes when internet gateway exist

### PR DESCRIPTION
**Pull request by mattkasun**
## Describe your changes
skip creation of explicit server & peer routes if internet gateway not present
## Provide link to Netmaker PR if required
n/a
## Provide testing steps
see GRA-1554
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [x] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [ ] Netclient & Netmaker are awesome.

### CODE REVIEW

File: routes/routes_linux.go
Comment (abhishek9686) [inline routes/routes_linux.go:1]: need these changes on other OS too

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: routes/routes_unix.go
Comment (abhishek9686) [inline routes/routes_unix.go:1]: what about check under func `SetNetmakerPeerEndpointRoutes` ?

File: routes/routes_windows.go
Comment (abhishek9686) [inline routes/routes_windows.go:1]: check under func `SetNetmakerPeerEndpointRoutes` is missing

**Review by abhishek9686 — APPROVED** [review]

**Review by afeiszli — APPROVED** [review]

**Comment by afeiszli** [comment]
Approved, but question for future sprint. Do we need a separate ticket to handle removing routes if gateway is removed?

**Comment by mattkasun** [comment]
> Approved, but question for future sprint. Do we need a separate ticket to handle removing routes if gateway is removed?

NET-36 created

**Pull request merged**